### ISSUE TITLE
The result on nuscenes test set is not good

### ISSUE THREAD

**Issue by zheng0819** [dvlab-research/VoxelNeXt#53]
Hi, thanks for your work, I met a question that the result on nuscenes validation set is as good as the pretrained model, but the result on nuscenes test set is even worth than that on nuscenes validation set. My implement is the same as that in VoxelNeXT paper, i.e. do not use  GT Sampling in last 5 epochs. But the result on nuscenes test set is not good. Could someone tell me why? Thanks!
